### DISCUSSION TITLE
Can perspective integrate with Django

### DISCUSSION THREAD

**Discussion by JalatorrS1** [perspective-dev/perspective#2540] [Q&A]
Hi guys , we were exploring perspective, and seems like an amazing visualization tool.  Is there any documentation or perhaps an example with an integration with Django? or perhaps special considerations we should take before running experiments? Some considerations if we were to implement DjangoHandler for channels?

**Comment by sinistersnare** [comment]
Hi there. Nothing special is needed for this integration. There are [some examples](https://github.com/finos/perspective/tree/master/examples) of using Perspective from a Python backend more generally. Perspective is primarily a frontend tool, which does not interact directly with the backend.
